plugins/graphiql/package.json

```diff
@@ -34,12 +34,13 @@
   "dependencies": {
     "@backstage/core-components": "workspace:^",
     "@backstage/core-plugin-api": "workspace:^",
+    "@backstage/plugin-catalog-react": "workspace:^",
     "@backstage/theme": "workspace:^",
     "@material-ui/core": "^4.12.2",
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "4.0.0-alpha.61",
     "@types/react": "^16.13.1 || ^17.0.0",
-    "graphiql": "^1.5.12",
+    "graphiql": "^1.8.8",
```

Comment:
was this needed? just checking, since mixing concerns adds to the risk of a revert of it all if something goes wrong

Reply:
to keep this consistent of the one used in api-docs